woodwork/utils.py

```diff
@@ -298,3 +299,149 @@ def _parse_logical_type(logical_type, name):
         return logical_type
     else:
         raise TypeError(f"Invalid logical type specified for '{name}'")
+
+
+def concat(objs, axis=1, join='outer', validate_schema=True):
+    """
+    Concatenate Woodwork objects along a particular axis. Cannot concatenate DataFrames
+    with conflicting Woodwork index or time index columns. If DataFrames have the same Woodwork
+    index or time index columns, the first appearance of the column will be included in the
+    concatenated DataFrame. As Woodwork does not allow duplicate column names,
+    will not allow duplicate columns at concatenation.
+
+    Args:
+        objs (list[Series, DataFrame]): The Woodwork objects to be concatenated. If Woodwork
+            is not initialized on any of the objects, type inference will be performed.
+        axis ({0/’index’, 1/’columns’}, optional): The axis to concatenate along. Defaults to 1.
+        join ({'inner', 'outer'}, optiona): How to handle indexes on other axis. Defaults to 'outer'.
+        validate_schema (bool, optional): Whether validation should be performed on the typing information
+            for the concatenated DataFrame. Defaults to True.
+
+    Returns:
+        object, type of objs: When objs contains at least one DataFrame, a Woodwork DataFrame is returned.
+            When concatenating along the columns (axis=1), a Woodwork DataFrame is returned.
+            When concatenating along the index (axis=1) with only series, a series is returned
+            without Woodwork initialized.
+    """
+    if not objs:
+        raise ValueError('No objects to concatenate')
+
+    # --> consider adding sort
```

Comment:
Adding `sort` as a parameter for Pandas DataFrames (and possibly Koalas, since the parameter is available there as well) means we might want to take into account `already_sorted` at woodwork init. 

I think it might make sense to revisit this in a separate issue, and we can add it if it's requested.

Reply:
My vote: delay this for now